src/new.rs

```diff
@@ -89,13 +98,24 @@ pub fn create_new_project<P: AsRef<Path>>(dest: P) -> Result<()> {
     Ok(())
 }
 
-pub fn create_new_document(doc_type: &str, name: &str, config: &Config) -> Result<()> {
+pub fn create_post(name: &str, config: &Config) -> Result<()> {
     let path = Path::new(&config.source);
-    let full_path = &path.join(&config.posts).join(name);
+
+    let local = Local::now().format("%d %b %Y %H:%M:%S %z").to_string();
+    let vector: Vec<u8> = Vec::from(new_post_tmpl);
+    let mut out = String::from_utf8(vector).unwrap();
+    out = out.replace("%title%", name);
+    out = out.replace("%date%", &local);
+    let new_post = &path.join(format!("{}/{}.md", &config.posts, name));
```

Comment:
Question: are we assuming `name` is a file name or a post name? 

If file name, then we should conditionally add `.md`.  Are we assuming the name is a slug?  Should we slugify it first and then titlize_slug for the `title` field?

If a post name, we should slugify the `name` when creating a path.

See https://github.com/cobalt-org/cobalt.rs/blob/master/src/slug.rs

Note: this question also applies to `create_new_document` and the `cobalt new` defaults.